### PR TITLE
Unicode, literally (and licenses too~)

### PR DESCRIPTION
**Pull request by akx**

### CODE REVIEW

File: _misc/ensure_unicode_literals.py [hunk 1]
```diff
@@ -42,24 +42,29 @@ def process_file(path):
...
+    need_encoding_comment = True
+    first_non_comment_line_index = 0
+    for line_index, line in enumerate(source_lines):
+        if not line.strip():
+            continue
+        if encoding_comment_regexp.match(line):
+            need_encoding_comment = False
+        if not line.startswith("#"):
+            first_non_comment_line_index = line_index
+            break
```
Comment (suutari-ai) [inline _misc/ensure_unicode_literals.py]: IMHO we should only install encoding comment headers to files that are not all ASCII.

Comment (akx) [inline _misc/ensure_unicode_literals.py]: Done and done.

Comment (suutari-ai) [inline _misc/ensure_unicode_literals.py]: i.e. `any(x > 127 for x in six.iterbytes(open(FILENAME, 'rb').read()))`

**Comment by suutari-ai** [comment]
Cool bits!

**Pull request merged**